### DISCUSSION TITLE
스테이지 구현 to do list 질문

### DISCUSSION THREAD

**Discussion by seong0929** [SeoulTechTCPGame/HumanSurvival#104] [Q&A]
- 초기화 적용이 무엇인가요?
- 스테이지에 횃불 자동스폰은 특정 위치나 주기 혹은 확률이 존재하나요?
- 횃불 에셋은 제가 사용한 에셋에서 활용하나요?
- [횃불 ](https://github.com/orgs/SUST22Unity/projects/2/views/1?pane=issue&itemId=17506292) to do list가 존재합니다.

**Comment by rapa1130** [comment]
### 초기화 적용이 무엇인가요?

[관련 문서 링크](https://github.com/SUST22Unity/HumanSurvival/wiki/%EB%A7%B5-%EC%8A%A4%ED%85%8C%EC%9D%B4%EC%A7%80-%EC%8B%9C%EC%8A%A4%ED%85%9C#%EC%8A%A4%ED%85%8C%EC%9D%B4%EC%A7%80)

매 게임을 시작할 때마다 초기화에 유의해야 하는 것들이 있습니다. 현재 체력, 지난 플레이 증감된 캐릭터 스탯, 게임 플레이 시간, 스킬/패시브 리스트, 캐릭터 위치, 캐릭터 레벨 및 경험치 와 같은 수치들은 게임 시작시 초기화 되어야 합니다. 따로 구현해야 되는 것은 아니고 확인만 한번 해주시면 되겠습니다.
 
### 스테이지에 횃불 자동스폰은 특정 위치나 주기 혹은 확률이 존재하나요?

네 확률은 존재합니다. 횃불 ToDo에 해당 부분이 미흡해 보강해보겠습니다.


### 횃불 에셋은 제가 사용한 에셋에서 활용하나요?
성혁님께서 사용하신 에셋이 무엇인지는 모르겠지만 제가 제공했던 [맵 오브젝트/아이템 에셋 링크](https://github.com/SUST22Unity/HumanSurvival/discussions/81)에서 이용하시길 추천 드립니다. 
![화로 대체 가능 이미지들](https://user-images.githubusercontent.com/49236828/215241425-62417294-7c83-423f-b222-b28882695bb2.PNG)
화로 이미지는 위 이미지들 중 하나로 대체 가능하다고 생각합니다.


###  [횃불](https://github.com/orgs/SUST22Unity/projects/2/views/1?pane=issue&itemId=17506292) to do list가 존재합니다.
맞습니다. 횃불 ToDoList 의 중복이라 보실 수도 있겠으나, 횃불 ToDoList는 횃불 랜덤 스폰 기능을 만드는 것이고 스테이지 ToDo에선 해당 기능을 적용하는 것입니다. 즉, 스테이지 ToDoList를 완료하기 위해선 횃불 ToDoList 가 선행되야 합니다. 하지만 이ToDo 는 제가 제공한 형식일 뿐 당장 횃불을 개발해야 한다거나 횃불 구현이 당장 필요할 것 같지 않으면 넘어가셔도 괜찮습니다. 

#### P.S.
추가적으로 스테이지 1에 대한 Wave 별 몬스터 정보와 등장 몬스터들의 정보를 오늘(2023-01-28)내로 정리할 예정입니다. 혹시 필요하시다면 참고바랍니다.
